### DISCUSSION TITLE
Who maintains Git?

### DISCUSSION THREAD

**Discussion by The-Futurist** [community/community#21714] [New to GitHub]
<p>It always seems a little mysterious when I try to find out more about who mainatains and improves Git itself.</p>
<p>Sure, I can see this gentleman is the named person:</p>
<p><a href="https://en.wikipedia.org/wiki/Junio_Hamano" rel="nofollow noopener">https://en.wikipedia.org/wiki/Junio_Hamano</a></p>
<p>and he has a blog - but not updated for three years:</p>
<p><a href="https://git-blame.blogspot.com/" rel="nofollow noopener">https://git-blame.blogspot.com/</a></p>
<p>then we have his GitHub page - but we can’t post issues:</p>
<p><a href="https://github.com/gitster/git" rel="nofollow noopener">https://github.com/gitster/git</a></p>
<p>then we have tha main Git repo - but again no issues can be posted:</p>
<p><a href="https://github.com/git/git" rel="nofollow noopener">https://github.com/git/git</a></p>
<p>note also that that repo has like 180 pending pull requests going back four years, most of which have stagnated and should be closed I guess, this doesn’t encourage participation by others. Then of course we have this:</p>
<p>“_The user discussion and development of Git take place on the Git <strong>mailing list</strong> – everyone is welcome to post bug reports, feature requests, comments and patches to <a href="mailto:git@vger.kernel.org">git@vger.kernel.org</a> (read <a href="https://github.com/git/git/blob/master/Documentation/SubmittingPatches" rel="nofollow noopener">Documentation/SubmittingPatches</a> for instructions on patch submission). To subscribe to the list, send an email with just “subscribe git” in the body to <a href="mailto:majordomo@vger.kernel.org">majordomo@vger.kernel.org</a>. The mailing list archives are available at <a href="https://public-inbox.org/git/" rel="nofollow noopener">https://public-inbox.org/git/</a>, <a href="http://marc.info/?l=git" rel="nofollow noopener">http://marc.info/?l=git</a>_<em>and other archival sites</em>.”</p>
<p>But a “mailing list” is rather antiquated and hardly the best way to handle feedback and feature requests for a technology that has become the backbone source code control system used across the globe.</p>
<p>Isn’t it time that Git itself opened up, became easier for developers to supply feedback and engage in conversations about Git technology?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/korporal">@korporal</a>,</p>
<p>As you’ve found out, git is maintained on the git mailing list. Junio is indeed the person shepherding the master and release branches, but there are many other contributors.</p>
<p>Discussions about the git development process do occur from time to time on that mailing list, you’re not the only one who thinks that a mailinglist is not the optimal way to develop git. Maybe have a look at the archives to understand both points of view. And if you have somthing new to add to that discussion, the mailinglist is open for everyone to subscribe.</p>
